backup.sh

```
@@ -86,118 +190,106 @@ while [[ $# -gt 0 ]]; do
 BACKUP_DIR="backup_$(date +%Y%m%d_%H%M%S)"
-ZIP_FILE="arcadia_backup_$(date +%Y%m%d_%H%M%S).zip"
+TIMESTAMP=$(date +%Y%m%d_%H%M%S)
 
-# Source configuration from backend .env files based on mode and priority
+# Source configuration from backend .env files
 if [ "$USE_DOCKER" = true ]; then
-    # Docker mode: prioritize .env.docker, fallback to .env
     if [ -f "backend/.env.docker" ]; then
         echo "Loading configuration from backend/.env.docker..."
-        export $(grep -v '^#' backend/.env.docker | grep -E '^(POSTGRES_|DB_|BACKUP_)' | tr -d '\r' | xargs)
+        export $(grep -v '^#' backend/.env.docker | grep -E '^(POSTGRES_|DB_|BACKUP_)' | tr -d '\r' | xargs 2>/dev/null) || true
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)